core/src/main/java/software/amazon/awssdk/runtime/auth/SignerProviderContext.java

```diff
@@ -15,21 +15,18 @@
 
 package software.amazon.awssdk.runtime.auth;
 
-import java.net.URI;
 import software.amazon.awssdk.RequestConfig;
 import software.amazon.awssdk.annotation.SdkProtectedApi;
 import software.amazon.awssdk.http.SdkHttpFullRequest;
 
 @SdkProtectedApi
 public class SignerProviderContext {
 
-    private final URI uri;
     private final boolean isRedirect;
```

Comment:
Do we need isRedirect either?

Reply:
We don't need the context at all anymore. Think we should save that cleanup for the signer refactor?

Reply:
+1, sounds good